### PR TITLE
Updated qtox formula and readme

### PR DESCRIPTION
**Pull request by fenque**

### CODE REVIEW

**Comment by stal888** [comment]
@fenque Does this work?

File: Formula/qtox.rb [hunk 1]
```diff
@@ -6,7 +6,8 @@ class Qtox < Formula
   
   depends_on "Tox/tox/libtoxcore"
   depends_on "qt5"
-  depends_on "homebrew/science/opencv"
++ depends_on "ffmpeg"
++ depends_on "openal-soft"
```
Comment (stal888) [inline Formula/qtox.rb]: suspicious

**Comment by fenque** [comment]
It works, qtox switched from opencv to ffmpeg, so i updated this formula, nothing suspicious.

**Comment by stal888** [comment]
I meant the extra plus signs. Bad merge?

**Comment by fenque** [comment]
And, lol, didn't notice, I'll fix that.

**Comment by stal888** [comment]
I'll take your word that it works.

Thanks for the pull

**Pull request merged**